Netkan/Sources/Curse/CurseApi.cs

```diff
@@ -26,18 +26,15 @@ public CurseMod GetMod(int modId)
 
             // Check if the mod has been removed from Curse and if it corresponds to a KSP mod.
             var error = JsonConvert.DeserializeObject<CurseError>(json);
-
-            if (error.code != 0)
-            {
-                var errorMessage = string.Format("Could not get the mod from Curse, reason: {0}.", error.message);
-                throw new Kraken(errorMessage);
-            }
-            else if (!error.game.Equals("Kerbal Space Program"))
+            if (!string.IsNullOrEmpty(error.error))
```

Comment:
Can we use `IsNullOrWhiteSpace` for an extra check for spaces?

Reply:
Sure.